samples/apps/github-qna-webapp-react/README.md

```diff
@@ -0,0 +1,30 @@
+# GitHub Repo Q&A Bot Sample
+
+> **!IMPORTANT**
+> This learning sample is for educational purposes only and should not be used in any
+> production use case. It is intended to highlight concepts of Semantic Kernel and not
+> any architectural / security design practices to be used.
+
+## Running the sample
+
+1. You will need an [Open AI Key](https://openai.com/api/) or
+   [Azure Open AI Service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart) for this sample.
+2. Ensure the service API is already running `http://localhost:7071`. If not learn how to start it [here](../starter-api-azure-function/README.md).
+3. **Run** the following command `yarn install` (if you have never run the sample before) and/or `yarn start` from the command line.
+4. A browser will open or you can navigate to `http://localhost:3000` to use the sample.
```

Comment:
Instructions on updating the keys would be useful, like, what's the difference between these two?

```
REACT_APP_AZURE_OPEN_AI_DEPLOYMENT=
REACT_APP_AZURE_OPEN_AI_COMPLETION_DEPLOYMENT=
```

Reply:
Also tough that it requires completion and embeddings be the same depoyment resource.

Reply:
Took care of this in a PR, thank you for catching!